components/menu/menu-item-selectable-styles.js

```diff
@@ -14,13 +14,6 @@ export const menuItemSelectableStyles = [ menuItemStyles,
 			text-decoration: none;
 		}
 
-		:host(:hover),
-		:host(:focus),
-		:host(:hover) > d2l-icon,
-		:host(:focus) > d2l-icon {
-			color: var(--d2l-color-celestine-minus-1);
-		}
```

Comment:
This is a slight change in behaviour so that the checkbox behaviour on hover/focus matches that of the chevron -- it also simplifies how theming would work.